ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.13__amd64-nightly-4.13-upgrade-from-aro.yaml

```diff
@@ -60,15 +50,12 @@ tests:
   cron: 6 16 2 * *
   steps:
     cluster_profile: azure-qe
+    env:
+      ARO_CLUSTER_VERSION: 4.13.23
```

Comment:
What is the default version when `ARO_CLUSTER_VERSION` is not set?

Reply:
current default is 4.12.25.